pkg/client/uuid.go

```diff
@@ -0,0 +1,273 @@
+package client
```

Comment:
If this was copied from pgtype, link to the original file + our PR maybe?

Reply:
Added a link to the file on Github.
Also added comments to the two places that changed `\\CQ-Change:`

Reply:
If this is not what you were looking for then let me know!